src/test/java/com/woowacamp/soolsool/core/liquor/service/LiquorServiceIntegrationTest.java

```diff
@@ -103,10 +104,13 @@ void increaseClick() {
     void increaseImpression() {
         // given
         long liquorId = 1L;
+        LiquorListRequest liquorListRequest = new LiquorListRequest(
+            null, null, null,
+            null, liquorId+1, null
```

Comment:
```suggestion
            null, liquorId + 1, null
```
Code style을 팀에서 사용하는 파일을 사용하는지 점검해볼 필요가 있을 것 같아요.